app.js

```diff
@@ -14,10 +15,36 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
-
 // Add the route handlers here:
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
+// Iteration 2_route endpoint home
+app.get('/', (req, res) => {
+  res.render('index.hbs');
+});
+
+//Iteration 3.1_route endpoint beers
+
+app.get('/beers', function (req, res) {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      console.log(beersFromApi);
```

Comment:
after you see the results you want in your terminal, remember to clean your console.logs!